src/github-social-provider.js

```diff
@@ -830,15 +828,15 @@ GithubSocialProvider.prototype.getUsers = function() {
  * Sends a message to another clientId.
  */
 GithubSocialProvider.prototype.sendMessage = function(toClientId, message) {
-  /*
   if (this.clientStates_[toClientId].status !== 'ONLINE') {
     return Promise.reject('Client not online');
   }
-  */
 
   var userId = this.clientStates_[toClientId].userId;
   var signalingGist = this.users_[userId].signaling;
-  return this.postComment_(signalingGist, MESSAGE_TYPES.MESSAGE, message, toClientId);
+  return this.postComment_(signalingGist, MESSAGE_TYPES.MESSAGE, message, toClientId).then(function() {
+    return Promise.resolve();
```

Comment:
Would this maybe work as `Promise.resolve(this.postComment_(...))` instead? Or if nothing else, I think .then(Promise.resolve) will probably work (instead of wrapping it in a declared function).